install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -113,6 +113,80 @@ value, triggers provisioning using the single implied
 link:#enabling-provisioner-plugins[provisioner plug-in per cloud].
 ====
 
+
+[[storage-provisioner-labels]]
+=== Storage-Provisioner Labels
+
+For dynamically provisioned storage, OpenShift defines three key/value pairs,
+called _storage-provisioner labels_ in this document,
+and associates them with the storage, in a storage-specific way.
+These are normally not visible to OpenShift users.
+Their main benefit is to enable storage administrators to recognize volumes
+associated with OpenShift.
+
+The term _storage-provisioner label_ is what this documentation uses.
+The actual term, that a storage administrator will understand, is listed
+in the following table:
+
+[cols="1,2",options="headers"]
+|====
+|Term for Key/Value Pairs
```

Comment:
"Term for Key/Value Pairs" and "Storage Type" are not showing up as bold headers here,  though the "Project Name/PVC Name" headers in the table below are behaving properly.